README.rst

```diff
@@ -103,61 +103,63 @@ Keyring Support
 ---------------
 
 Instead of typing in your password every time you upload a distribution, Twine
-allows you to store your username and password securely using `keyring`_.
-
-To use the keyring, you must first install the keyring packages:
-
-- On Windows and MacOS you just need to install ``keyring``, for example,
-  ``pip install --user keyring``.
-- On Linux, in addition to the ``keyring`` package you also need to ensure the
-  ``python3-dbus`` system package is installed. For example, ``apt install
-  python3-dbus``. See `Keyring's installation instructions`_ for more details.
+allows storing a username and password securely using `keyring`_.
+Keyring is installed with Twine but for some systems (Linux mainly) may
+require additional installation steps; refer to the keyring project page
+for details.
 
-Once keyring is installed you can use the ``keyring`` program to set your
-username and password to use for each package index (repository) you want to
-upload to using Twine.
+Once Twine is installed, use the ``keyring`` program to set a
+username and password to use for each package index (repository) to
+which you may upload.
 
-To set your username and password for Test PyPI run the following command.
-``keyring`` will prompt you for your password:
+For example, to set a username and password for ``$REPOSITORY``,
```

Comment:
I kind of liked the concrete example of Test PyPI. What do you think about changing it to use `https://upload.pypi.org/legacy/`, with instructions below for replacing that with `https://test.pypi.org/legacy/`?

I also don't feel strongly about this. :)

Reply:
I also like your suggested approach. I just wanted to avoid the somewhat awkward duplication.

Reply:
Done in 5d0b25d.